README.md

```diff
@@ -337,20 +336,7 @@ Optionally, anonymous access can be turned on. In this case an extra `anonymous`
 
 ### Importing comments
 
-Remark supports importing comments from Disqus, WordPress or native backup format.
-All imported comments has `Imported` field set to `true`.
-
-#### Initial import from Disqus
-
-1.  Disqus provides an export of all comments on your site in a g-zipped file. This is found in your Moderation panel at Disqus Admin > Setup > Export. The export will be sent into a queue and then emailed to the address associated with your account once it's ready. Direct link to export will be something like `https://<siteud>.disqus.com/admin/discussions/export/`. See [importing-exporting](https://help.disqus.com/customer/portal/articles/1104797-importing-exporting) for more details.
-2.  Move this file to your remark42 host within `./var` and unzip, i.e. `gunzip <disqus-export-name>.xml.gz`.
-3.  Run import command - `docker exec -it remark42 import -p disqus -f /srv/var/{disqus-export-name}.xml -s {your site id}`
-
-#### Initial import from WordPress
-
-1. Install WordPress [plugin](https://wordpress.org/plugins/wp-exporter/) to export comments and follow it instructions. The plugin should produce a xml-based file with site content including comments.
-2. Move this file to your remark42 host within `./var`
-3. Run import command - `docker exec -it remark42 import -p wordpress -f {wordpress-export-name}.xml -s {your site id}`
+Please see https://remark42.com/docs/latest/import/
```

Comment:
I don't think this is a good idea to make readme navigate to docs on the official site. maybe link to docs/latest/import.md  or smth like this?